src/os_sensor_nodelet.cpp

```diff
@@ -287,12 +293,33 @@ class OusterSensor : public OusterClientBase {
             config_flags |= ouster::sensor::CONFIG_UDP_DEST_AUTO;
         }
 
+        if (sensor::in_multicast(udp_dest)) {
```

Comment:
Unfortunately the method `sensor::in_multicast` doesn't handle the case when `udp_dest` is empty string, so it is necessary to add the following check
```suggestion
        if (is_arg_set(udp_dest) && sensor::in_multicast(udp_dest)) {
```